config/default/metrics_service.yaml

```diff
@@ -7,9 +7,9 @@ metadata:
   namespace: system
 spec:
   ports:
-  - name: https
-    port: 8443
+  - name: http
```

Comment:
What’s the reason to change it to insecure connection?

Reply:
@sonam1412 , following to the [thread](https://redhat-internal.slack.com/archives/C034UFMUSV6/p1744903328634849) as @dirgim shared how release made these changes and seems it is working for them